resources/client/cl_main.ts

```diff
@@ -1,13 +1,13 @@
 import { sendMessage } from '../utils/messages';
 import { PhoneEvents } from '../../typings/phone';
 import { TwitterEvents } from '../../typings/twitter';
-import { phoneCloseAnim, phoneOpenAnim, removePhoneProp } from './functions';
+import { removePhoneProp } from './functions';
```

Comment:
Why is this imported here? I only see its usage in `animation.service.ts`

Reply:
This was still here because it was used onResourceStop, though its still better to just convert it to use the animation service